inst/app/modules/main_map.R

```diff
@@ -1,20 +1,21 @@
 
-# Get information of all types of vehicles involved in the accidents
+# Get information of all types of vehicles involved in the accidents to show in popup
 # TODO: prepare following mutated dataset before running shiny to save loading time?
 hk_vehicles_involved = hk_vehicles %>%
```

Comment:
Apply consistent assignment operator.
```suggestion
hk_vehicles_involved <- hk_vehicles %>%
```